src/components/data-exploration/CellInfo.jsx

```diff
@@ -57,7 +44,7 @@ const CellInfo = (props) => {
     </Card>
   );
 
-  if (cellInfoVisible && cellInfo.cellId && Object.keys(coordinates.current).length > 0) {
+  if (cellInfo.cellId && Object.keys(coordinates.current).length > 0) {
```

Comment:
Is there a better way to check if `coordinates` exists instead of checking for `Object.keys(coordinates.current).length`? Does the `coordinates.current` not contain anything when we're not hovering?

Reply:
I removed the whole check because we don't need it anymore